R/utilities-task.R

```diff
@@ -640,3 +644,37 @@ loadPlotPKRatioTask <- function(workflow, configurationPlan) {
     settings = list(axes = getAxesProperties(configurationPlan$plots$AxesSettings$PKRatioPlots))
   ))
 }
+
+
+#' @title loadPlotDDIRatioTask
+#' @description
+#' Define `plotDDIRatio` task and its settings
+#' @param workflow `QualificationWorkflow` object
+#' @param configurationPlan A `ConfigurationPlan` object
+#' @return A `QualificationTask` object
+loadPlotDDIRatioTask <- function(workflow, configurationPlan) {
+  validateIsOfType(workflow, "QualificationWorkflow")
+  validateIsOfType(configurationPlan, "ConfigurationPlan")
+
+  # Time Profiles task is only active if the field is defined & not empty
+  active <- !isOfLength(configurationPlan$plots$DDIRatioPlots, 0)
+
+  taskFunction <- plotQualificationDDIs
+  nameFunction <- deparse(substitute(plotQualificationDDIs))

```

Comment:
`deparse(substitute` is everywhere... It's not like I don't like it, it's just that I have no idea what it does. Can we create a method that actually has a name that we understand, also in 6 months from now?

Reply:
merged these into one function, getObjectNameAsString